app/api/helpers/csv_jobs_util.py

```diff
@@ -17,3 +17,26 @@ def export_orders_csv(orders):
             rows.append(column)
 
     return rows
+
+
+def export_attendees_csv(attendees):
+    headers = ['Order#', 'Order Date', 'Status', 'First Name', 'Last Name', 'Email',
+               'Country', 'Payment Type', 'Ticket Name', 'Ticket Price', 'Ticket Type']
+
+    rows = [headers]
+    for attendee in attendees:
+        column = [str(attendee.order.get_invoice_number()) if attendee.order else '-',
```

Comment:
can we abstract this using a function with `try` like 
```python
safe_str(attendee.order.get_invoice_number(), '-')`
```
?

Reply:
Good idea! I will do that